src/app/map/components/time-slider/time-slider.component.html

```diff
@@ -27,23 +27,23 @@
             (dateChange)="selectedDatePickerDate($event.value, true)"
             [matDatepicker]="minDatePicker"
             [min]="availableDates[0]"
-            [max]="availableDates[effectiveMaximumDateIndex - 1]"
+            [max]="availableDates[maximumDateIndex - 1]"
           />
           <mat-datepicker
             #minDatePicker
             [startView]="datePickerStartView"
-            [startAt]="availableDates[effectiveMinimumDateIndex]"
+            [startAt]="availableDates[this.firstSliderPosition]"
```

Comment:
This desing desicion, but I would say the datepicker should open at the value of the current range, since this is what we are changing. (We are not changing the selected date, only the range, see other comment)
Also, there is no need to use `this` in the template

```suggestion
            [startAt]="availableDates[minimumDateIndex]"
```

Reply:
Current state is preferred. Must be specifically evaluated again during future timeslider revision.

Reply:
You can still remove `this`

Reply:
I removed `this` from line 35 and 61 in `src/app/map/components/time-slider/time-slider.component.html`